_data/projects/EigengrausGenerator.yml

```
@@ -12,8 +12,8 @@ tags:
 - generation
 - tabletop
 upforgrabs:
-  name: good first issue
-  link: https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator/labels/good%20first%20issue
+  name: help wanted
+   link: https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator/labels/help%20wanted
```

Ugh, I didn't notice this whitespace change I introduced. Apologies:

```suggestion
  link: https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator/labels/help%20wanted
```